ticdc/ticdc-faq.md

```diff
@@ -266,7 +266,19 @@ When upstream write traffic is at peak hours, the downstream may fail to consume
 
 ## Why does replication using TiCDC stall or even stop after data restore using TiDB Lightning and BR from upstream?
 
-Currently, TiCDC is not yet fully compatible with TiDB Lightning and BR. Therefore, please avoid using TiDB Lightning and BR on tables that are replicated by TiCDC.
+Currently, TiCDC is not yet fully compatible with TiDB Lightning and BR. Therefore, avoid using TiDB Lightning and BR on tables that are replicated by TiCDC. Otherwise, unknown errors might occur, such as TiCDC replication getting stuck, a significant spike in replication latency, or data loss.
+
+If you need to use TiDB Lightning or BR to restore data for some tables replicated by TiCDC, take these steps:
+
+1. Delete the TiCDC replication task related to these tables.
+
+2. Use TiDB Lightning or BR to restore data separately in the upstream and downstream clusters of TiCDC.
+
+3. After the restoration is complete and data consistency between the upstream and downstream clusters is verified, create a new TiCDC replication task for incremental replication, with the timestamp (TSO) from the upstream backup as the `start-ts` for the task. For example, assuming the snapshot time point of the BR backup in the upstream cluster is `431434047157698561`, you can create a new TiCDC replication task using the following command:
+
+   ```shell
+   cdc cli changefeed create -c "upstream-to-downstream-some-tables" --start-ts=431434047157698561 --sink-uri="mysql://root@127.0.0.1:4000? time-zone="
+   ```
```

Comment:
````suggestion
    ```shell
    cdc cli changefeed create -c "upstream-to-downstream-some-tables" --start-ts=431434047157698561 --sink-uri="mysql://root@127.0.0.1:4000? time-zone="
    ```
````